Automattic/woocommerce-payments
# Reporting: Show bank reference ID on Payout details page

Fixes #9878 

#### Changes proposed in this Pull Request

* Add an additional payout details header to show `Bank account details` and `Bank reference ID` for any payout.

![image](https://github.com/user-attachments/assets/81f6f3dd-afd0-44c2-bef0-e3c57f8d5bc5)

* The PR introduces a `Copy button` component to allow easy click-and-copy of any text value passed on to it.  

![image](https://github.com/user-attachments/assets/8a342bf2-ef9e-4228-81fb-50a073b632d9)


#### Testing instructions

* Make sure your local test site is connected to an account with a few payouts - preferably a combination of failed, successful and instant payouts. You may connect to `acct_1OuYn1QpUShfetUA`  if you like.
* Checkout this branch and rebuild CSS and assets
* Browse to `Payments > Payouts` and click any payout to get to its details page. 
* You should be able to see an additional Payout details header similar to the screenshot above
* If it is a successful payout, you may also see a Bank Reference ID. Click the copy icon and test if the ID gets copied correctly
* For instant deposits, failed payouts, and when not available, the Bank reference key will show as `Not available`
* Test this over different screen sizes to see how the display adapts to mobile and tablet screen proportions

-------------------


**Post merge**
